backend/src/src-vulnerabilities/src/main/java/org/eclipse/sw360/vulnerabilities/VulnerabilityHandler.java

```diff
@@ -301,7 +309,9 @@ public RequestStatus updateReleaseVulnerabilityRelation(ReleaseVulnerabilityRela
                 ? user.getSecondaryDepartmentsAndRoles().entrySet().stream().flatMap(entry -> entry.getValue().stream())
                         .collect(Collectors.toSet())
                 : new HashSet<UserGroup>();
-
+        if (!(UserGroup.SECURITY_ADMIN == user.userGroup || UserGroup.ADMIN == user.userGroup || UserGroup.SW360_ADMIN == user.userGroup)){
```

Comment:
Use PermissionUtils class as suggested in your previous PR(update vuls of component)